WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3177,4 +3174,12 @@ extension AztecPostViewController {
         static let acceptTitle  = NSLocalizedString("Yes", comment: "Yes")
         static let cancelTitle  = NSLocalizedString("Not Now", comment: "Nicer dialog answer for \"No\".")
     }
+
+    struct UnsavedChangesAlert {
```

Comment:
I'm not a big fan of namespacing with structs and clases (remember `Libxml2`) because structs and clases can be instantiated (and instantiating `UnsavedChangesAlert` wouldn't make much sense as it is).

Considering these strings only make sense within `showPostHasChangesAlert()`, why not just instantiate these as local constants there?

ie: it doesn't feel like the hold much value outside of that method.